hack/utils.sh

```diff
@@ -25,3 +25,27 @@ cd_root_path() {
     cd "$(get_root_path)" || exit
 }
 
+# get_capd_root_path returns the root path of CAPD source tree
+get_capd_root_path() {
```

Comment:
nit: i don't think we need the capd functions

Reply:
I thought we would just need to copy the whole file, I can amend.